integration/experiment/system_characterization/sys_charac_freq_sweeps.sh

```diff
@@ -3,11 +3,12 @@
 #  SPDX-License-Identifier: BSD-3-Clause
 #
 
+module load geopm-runtime
 export GEOPM_SOURCE="${GEOPM_SOURCE?-Set GEOPM_SOURCE to full path of file with initialization parameters}"
+export PYTHONPATH=${GEOPM_SOURCE}/:${GEOPM_SOURCE}/geopmpy/:${GEOPM_SOURCE}/geopmdpy/:${HOME}/.local/lib/python3.6/site-packages/:$PYTHONPATH
```

Comment:
Why is this necessary?
geopmdpy and geopmpy should either be installed through pip as the user (in which case they'd end up in ~/.local) or module loaded if available.

Reply:
TODO: just rely on the user to have  geopm built. 
Ref: https://github.com/geopm/geopm/tree/dev/integration/test/plan#development-snapshot